Translations/pl.po

```diff
@@ -2389,22 +2362,23 @@ msgstr "F5, F6, F7, F8:     Ustawianie skrótu klawiszowego umiejętności"
 
 #: Source/help.cpp:44
 msgid "Shift + Left Mouse Button: Attack without moving"
-msgstr "Shift + LPM: Atak w miejscu"
+msgstr "Shift + Lewoklik: Atak w miejscu"
```

Comment:
LPM was ok in my opinion

Reply:
Then I'd actually do "Lewy/prawy przycisk myszy", as my friend said "lewoklik" sounds like some disease and it's also not a word I'm familiar with